examples/9_short_fiber_orientation_tensor.py

```diff
@@ -0,0 +1,117 @@
+"""
+.. _short_fiber_orientation_tensor:
+
+Plot of the orientation tensor for short fiber composites
+---------------------------------------------------------
+
+This example shows how to reconstruct and plot the components
+of the fiber orientation tensor in the global coordinate system.
+
+For more details about the fiber orientation tensor,
+refer to the Short Fiber Composites help.
+
+"""
+# %%
+# Set up analysis
+# ~~~~~~~~~~~~~~~
+# Setting up the analysis consists of loading Ansys libraries, connecting to the
+# DPF server, and retrieving the example files.
+#
+# Load Ansys libraries.
+
+import ansys.dpf.core as dpf
+import numpy as np
+from scipy.spatial.transform import Rotation
```

Comment:
Note that scipy is installed only for the doc build. A user that just installs pydpf-composites does not get scipy and will have to install scipy. I think it is good that scipy is not a dependency of pydpf-composites, but it probably makes sense to add a comment here.

Reply:
Agree, thanks. Added a note.